### PR TITLE
add quick start for application CRD

### LINKED ISSUE #1054: [Feature] Quick Start for App CRD
Since now we've have App CRD online, we should replace the existing Quick Start guide with App CRD. The existing one will be moved to End User docs.

And all End User docs will finally be replaced by new vela control plane doc.

### PR DESCRIPTION
**Pull request by wonderflow**
fix #1054

### CODE REVIEW

File: docs/en/quick-start.md [hunk 1]
```diff
@@ -13,50 +9,65 @@ Make sure you have finished and verified the installation following [this guide]
...
-    Type: webservice
-    HEALTHY Ready: 1/1
-    Last Deployment:
-      Created at: ...
-      Updated at: ...
-    Traits:
-      - ✅ ingress: Visiting URL: testsvc.example.com, IP: <your IP address>
+Under the neath, the K8s resources was created:
+
+```bigquery
```
Comment (hongchaodeng) [inline docs/en/quick-start.md:54]:
```suggestion
```bash
```

File: docs/en/quick-start.md [hunk 2]
```diff
@@ -75,4 +86,4 @@ Hello World
 
 ## What's Next
 
-- Learn details about [`Appfile`](./developers/learn-appfile.md) and know how it works.
+- Learn details about [`Application`](./application.md) and know how it works.
```
Comment (hongchaodeng) [inline docs/en/quick-start.md:89]:
```suggestion
- Learn more details about [`Application`](./application.md) and understand how it works.
```

File: docs/examples/vela-app.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      settings:
+        image: crccheck/hello-world
+        port: 8000
+      traits:
+        - name: ingress
+          properties:
+            domain: testsvc.example.com
+            http:
+              "/": 8000
+      type: webservice
```
Comment (hongchaodeng) [inline docs/examples/vela-app.yaml:17]: put `type` under `name`.

**Review by hongchaodeng — APPROVED** [review]
lgtm after nit

**Review by resouer — APPROVED** [review]

**Comment by resouer** [comment]
Kinda Off-topic but another missing doc is a guide to test/dry-run the CUE templates. Could be part of this one: https://kubevela.io/#/en/cue/basic

**Comment by wonderflow** [comment]
> https://kubevela.io/#/en/cue/basic

Can we just refer to this link?

**Comment by resouer** [comment]
@wonderflow I mean the current https://kubevela.io/#/en/cue/basic is lacking a guide to test/dry-run CUE templates.

**Pull request merged**